.nvmrc

```diff
@@ -0,0 +1 @@
+v20
```

Comment:
Is this required? Is node required for something?

Reply:
Yes, for this action:  `brown-ccv/alex-recommends`